src/om_pouch/core.cljs

```diff
@@ -118,8 +118,10 @@
           query (get-in pouchput [:params])]
       
       (let [id (:_id query)
-            nitem {:item/by-id {id {:_id id :item "I JUST CHANGED YOU"}}}]
-        (cb nitem pouchput))))
+            stable-id (random-uuid)
+            result {'pouchput {:tempids {[:item/by-id id] [:item/by-id stable-id]}}
+                    [:item/by-id stable-id] {:_id stable-id :item (str (:item query) " - I JUST CHANGED YOU")}}]
+        (cb result))))
```

Comment:
hey @cmcfarlen this seems to merge the new item into the state, but the list does not automatically re-render the item + "I JUST CHANGED YOU" string. If I do something that calls for a re-render, the state shows with the text above. Is it working like that on your end too?